simulator/src/main/java/com/hazelcast/simulator/protocol/core/ProtocolUtil.java

```diff
@@ -0,0 +1,49 @@
+package com.hazelcast.simulator.protocol.core;
+
+import java.util.Random;
+import java.util.concurrent.atomic.AtomicLong;
+
+public final class ProtocolUtil {
+
+    private static final String MESSAGE_DATA = "{\"testId\":\"StringStringMapTest\"}";
+
+    private static final int MAX_ADDRESS_INDEX = 4;
+    private static final AddressLevel MIN_ADDRESS_LEVEL = AddressLevel.AGENT;
+
+    private static final int MIN_ADDRESS_LEVEL_VALUE = MIN_ADDRESS_LEVEL.getInt();
+
+    private static final Random RANDOM = new Random();
+    private static final AtomicLong MESSAGE_ID = new AtomicLong();
+
+    private ProtocolUtil() {
+    }
+
+    public static SimulatorMessage buildRandomMessage() {
```

Comment:
I moved this to the main folder next to the standalone main classes, so it can be spotted easier as a delete candidate (if not used by tests later).